README.md

```diff
@@ -146,7 +152,26 @@ vignette("introduction-ospsuite")
 
 # Known issues
 
-**RStudio crashes when trying to load a workspace.** The ospsuite package uses the features implemented in PK-Sim and MoBi by creating `.NET` objects (e.g. a simulation) and using them from R. These objects cannot be saved as part of the workspace and reloaded on next start. When trying to do so, RStudio simply crashes. There is no possibility to overcome this limitation. To prevent RStudio from crashing, make sure to disable the check-box "Restore `.RData` into workspace at startup" in the options of RStudio. Keep in mind that you can also change this setting for specific projects.
+
+- Loading `ospsuite` might fail if your systems locale is not set to *English*, e.g.: 
+
+```ibrary(ospsuite)
+载入需要的程辑包：rClr
+Loading the dynamic library for Microsoft .NET runtime...
+Loaded Common Language Runtime version 4.0.30319.42000
+
+Error: package or namespace load failed for ‘ospsuite’:
+loadNamespace()里算'ospsuite'时.onLoad失败了，详细内容：
+调用: rClr::clrCall(dimensionTask, "AllAvailableUnitNamesFor", enc2utf8(dimension))
+错误: Type: System.Collections.Generic.KeyNotFoundException
+Message: Dimension 'CV mmHg*s虏/ml' not available in DimensionFactory.
+...
+```
+
+On Windows, set `Settings > Language > Administrative language settings > Current language for non-Unicode programs` 
+to `English (United States)` and reboot.
```

Comment:
on Linux: set the environment variable **LC_ALL** before starting R:
`export LC_ALL=en_US.UTF-8`

Reply:
Added

Reply:
Added